gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -689,20 +689,50 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     }
     case GTT_STAND:
     {
-      gchar **strv = NULL;
-
-      strv = g_strsplit (filter->option, ":", -1);
+      gchar **str_options = NULL;
+      guint i, str_options_num;
 
-      filter->data_stand.mode = gst_tensor_transform_get_stand_mode (strv[0]);
       filter->data_stand.out_type = _NNS_END;
-      if (g_strv_length (strv) > 1)
-        filter->data_stand.out_type = gst_tensor_get_type (strv[1]);
+      filter->data_stand.per_channel = FALSE;
 
-      g_strfreev (strv);
+      str_options = g_strsplit (filter->option, ",", -1);
+      str_options_num = g_strv_length (str_options);
+
+      for (i = 0; i < str_options_num; i++) {
+        gchar **options_strv = g_strsplit (str_options[i], ":", -1);
+
+        if (g_strv_length (options_strv) == 1) {
+          if (g_ascii_strcasecmp (options_strv[0], "default") == 0) {
```

Comment:
plz revert this, below is allowed
```
tensor_transform mode=stand option=default
tensor_transform mode=stand option=default:<TYPE>
tensor_transform mode=stand option=dc-average
tensor_transform mode=stand option=dc-average:<TYPE>
```

For the case g_ascii_strcasecmp (options_strv[0], "default") ==0 or g_ascii_strcasecmp (options_strv[0], "dc-average") == 0, 
if g_strv_length (options_strv) is 1, then options_strv[1] is null and below code is ok.
If necessary, it is better to add conditions:
```
if (g_strv_length (options_strv) > 1)
  filter->data_stand.out_type = gst_tensor_get_type (options_strv[1]);
```

Reply:
Thanks. I updated the code.